RileyLinkKit/PumpOpsSynchronous.swift

```diff
@@ -62,18 +62,47 @@ class PumpOpsSynchronous {
         return message
     }
     
+    /**
+     Attempts to send initial short wakeup message that kicks off the wakeup process.
+
+     Makes multiple attempts to send this message in order to work around issue w/
+     x22 pump/RileyLink combos not responding to the initial short wakeup message.
+
+     If successful, still does not fully wake up the pump - only alerts it such that the
+     longer wakeup message can be sent next.
+     */
+    private func attemptShortWakeUp(attempts: Int = 3) throws {
+        var lastError: ErrorType?
+
+        for attempt in 0..<attempts {
+            do {
+                let shortPowerMessage = makePumpMessage(.PowerOn)
+                let shortResponse = try sendAndListen(shortPowerMessage, timeoutMS: 15000, repeatCount: 255, msBetweenPackets: 0, retryCount: 0)
+
+                if shortResponse.messageType == .PumpAck {
+                    // Pump successfully received and responded to short wakeup message!
+                    return
+                } else {
+                    lastError = PumpCommsError.UnknownResponse("Wakeup shortResponse: \(shortResponse.txData)")
+                }
+            } catch let error {
+                print("Wakeup failure on attempt #\(attempt): \(error)")
```

Comment:
This is probably unnecessary logging, since it's not something we previously logged.

Eventual stretch goal would be to remove all direct NSLog calls and let the framework's client choose how to handle logging, but that's a long way off.